src/theme/components/textInput.ts

```diff
@@ -0,0 +1,53 @@
+const activeFocus = {
```

Comment:
We need to make this a global style object that we can use everywhere, similar to the mixin that was used in the SCSS.

Reply:
Yes, there are other styles that should be global styles that popped up in other PRs. I'll make a ticket to create a `global.ts` file.

Reply:
OK, great.  I also want to have a discussion about creating a master NYPL Chakra theme file.